hypervisor/lib/crypto/mbedtls/sha256.c

```diff
@@ -110,64 +106,65 @@ int32_t mbedtls_sha256_starts_ret( mbedtls_sha256_context *ctx, int32_t is224 )
 
 static const uint32_t K[] =
```

Comment:
it is better to use lower case for variable name

Reply:
I'd like to keep it.